### PR TITLE
Update script-topics.rpy   How much do you love me?

### PR DESCRIPTION
**Pull request by todemodu**
added a love topic where Monika can tell the player how much she loves them.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -5616,3 +5616,15 @@ label monika_dreaming:
     m 1a "But can you just imagine what it would be like to meet me in your dreams [player], any night that you wanted to?"
     m 1j "I hope that someday we can make your dreams about us a reality, my love."
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_i_love_you" ,category=['romance'],prompt="how much do you love me?",random=True))
```
Comment (Olliesama) [inline Monika After Story/game/script-topics.rpy]: While it's completely fine this way, it right now occurs completely randomly as opposed to being an option for the player to ask her as a question.
If you want it to act as question and not a repeatable conversation, change `random=True` to `pool=True`

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -5616,3 +5616,15 @@ label monika_dreaming:
...
+
+label monika_i_love_you:
+    m 1f "[player], I love you."
+    m 1f "with all my heart."
```
Comment (Olliesama) [inline Monika After Story/game/script-topics.rpy]: The expression doesn't need to be repeated if used previously.
There are also several lines that require capitalization.

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -5616,3 +5616,15 @@ label monika_dreaming:
...
+    m 2g "you know that right?"
```
Comment (Olliesama) [inline Monika After Story/game/script-topics.rpy]: Capitalization.

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -5616,3 +5616,15 @@ label monika_dreaming:
...
+    m 3h "I dont know what I'd do without you."
+    m 5 "You mean so much to me."
+    m 1e "please, never forget that."
```
Comment (Olliesama) [inline Monika After Story/game/script-topics.rpy]: And here too.

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -5616,3 +5616,15 @@ label monika_dreaming:
...
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_i_love_you" ,category=['romance'],prompt="how much do you love me?",random=True))
+
+label monika_i_love_you:
+    m 1f "[player], I love you."
+    m 1f "with all my heart."
+    m 2g "you know that right?"
+    m 3h "I dont know what I'd do without you."
+    m 5 "You mean so much to me."
```
Comment (Olliesama) [inline Monika After Story/game/script-topics.rpy]: If you're intending the do the expression where Monika leans in then it's `5a` not `5`.
I'd also not use it unless you're ending with it or Monika has a sudden new train of thought.

**Comment by Subzerosan** [comment]
Sorry but this will have to be rejected

There's already a similar topic.

https://github.com/Monika-After-Story/MonikaModDev/pull/915

**Pull request closed without merging**